src/server/game/Entities/Unit/Unit.h

```diff
@@ -1986,6 +2053,7 @@ class TC_GAME_API Unit : public WorldObject
         VisibleAuraMap m_visibleAuras;
 
         float m_speed_rate[MAX_MOVE_TYPE];
+        float collisionHeight; // @todo: initialize this value using dbc data at unit creation
```

Comment:
yes. It's still a 'todo'. `collisionHeight` is only initialized when the player gets for the first time (of the game session) on a mount. Until then, `GetCollisionHeight()` returns 0.0. It's a non issue since this method is not used at the moment. But in the future, ComputeCollisionHeight() should be moved to the Unit class. And `collisionHeight` should be used for pathfinding (it represents the physical height of the unit).